test/math.jl

```diff
@@ -1008,7 +1008,11 @@ using .Main.Furlongs
         @test @inferred hypot(i, i, i, i) ≈ 2.0i
         @test @inferred hypot(i//1, 1//i, 1//i) ≈ i
     end
-    @test @inferred hypot(typemin(Int), typemin(Int)) ≈ -sqrt(2)typemin(Int)
+    let i = typemin(Int)
+        @test @inferred hypot(i, i) ≈ -√2i
+        @test @inferred hypot(i, i, i) ≈ -√3i
+        @test @inferred hypot(i, i, i, i) ≈ -2.0i
```

Comment:
These are the wrong answers — the result can't be negative.

Reply:
It is not. Note that `typemin(Int)` is negative.